pynest/examples/pynest-microcircuit/sim_params.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+#
+# sim_params.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+'''
+pynest microcicuit simulation parameters
+----------------------------------------------
+
+Simulation Parameters for the microcircuit.
+
+Hendrik Rothe, Hannah Bos, Sacha van Albada; May 2016
+'''
+
+import os
+current_dir = os.getcwd()
+full_path = current_dir + '/data/'
+sim_dict = {
+    # Simulation time (in ms)
+    't_sim': 1000.0,
+    # sim_resolution = Resolution of the simulation (in ms)
```

Comment:
Please make the explanations consistent, so remove the 'sim_resolution = ' here.